src/connector.ts

```diff
@@ -134,4 +142,20 @@ export class Connector {
 
 		addedChanged(unprotectString(this.coreHandler.core.deviceId))
 	}
+
+	private setupReloadDataKoaEndpoint(): void {
+		this.koaRouter.get('/reloadData/:rundownName', async (ctx, next): Promise<void> => {
+			const RUNDOWN_EXTERNAL_ID_SUFFIX = '_1'
+			if (!this.iNewsFTPHandler.iNewsWatcher) {
+				return
+			}
+			this.iNewsFTPHandler.iNewsWatcher?.ResyncRundown(ctx.params.rundownName + RUNDOWN_EXTERNAL_ID_SUFFIX)
```

Comment:
Do you need the `?`? You just checked that you had an `iNewsWatcher` on line 149.
If you still need it, then you shouldn't need the if-statement.

Reply:
A mistake on my part, i must have forgot to remove it after i made the guard for that very purpose :-)

Reply:
I have changed it!